plugin/src/test/resources/com/google/errorprone/xplat/checker/testdata/LegacyTimeBanNegativeCases.java

```diff
@@ -0,0 +1,51 @@
+package com.google.errorprone.xplat.checker.testdata;
+
+import com.google.errorprone.xplat.checker.AllowLegacyTime;
+
+import java.sql.Time;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.TimeZone;
+import java.util.SimpleTimeZone;
+
+public class LegacyTimeBanNegativeCases {
+
+  // fields
+  @AllowLegacyTime
+  private Calendar cal;
+  @AllowLegacyTime
+  private Date date;
+  @AllowLegacyTime
+  private GregorianCalendar calG;
+  @AllowLegacyTime
+  private TimeZone timeZone;
+  @AllowLegacyTime
+  private SimpleTimeZone simpleTimeZone;
+
+  // constructor param
+  public LegacyTimeBanNegativeCases(@AllowLegacyTime Calendar cal, @AllowLegacyTime Date date) {
+    this.cal = cal;
+  }
+
+  // method
```

Comment:
Add another example that calls a "legacy" API here, such as `calParam(Calendar)` below, and add something like `@AllowLegacyTime("required by legacy API")` as a self-documenting annotation usage example?